src/zzz_od/application/world_patrol/operation/world_patrol_run_route.py

```
@@ -311,12 +311,20 @@ def _update_current_pos(self, mini_map: MiniMapWrapper) -> Point | OperationRoun
     def _is_next_pos_valid(self, next_pos: Point) -> bool:
         """
         判断匹配的下一个坐标是否合法
-        1. 根据上一坐标、朝向、转向等 判断当前坐标计算结果是否偏离方向
+        1. 距离检查：防止基准点错误导致的大幅跳跃
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)